packages/CopyInput/src/CopyInput.js

```diff
@@ -100,18 +112,25 @@ const propTypes = {
   isReadOnly: PropTypes.bool,
   /** Default value for the input */
   value: PropTypes.string,
+  /** Should hide the value text */
+  hasValueShown: PropTypes.bool,
+  /** Is the input component used */
+  hasInputContainer: PropTypes.bool,
 };
 
 const defaultProps = {
   children: null,
   isReadOnly: true,
   value: "",
+  hasValueShown: true,
```

Comment:
While the result is correct, it's a bit confusing that we only show the "value" label if `hasInputContainer==false`.  I feel like we should have this new prop, `hasValueShown` as `false` by default, and leave it to the consumer not to set both to toggle each option independently.  A little less convenient, but also easier to understand (and your logic above will be easier to follow/maintain).